src/components/templates/modal/Modal.jsx

```diff
@@ -0,0 +1,18 @@
+
+import React from 'react'
+
+function Modal ({children, showModal}) {
+  return ( 
+    <>
```

Comment:
```suggestion
    <Fragment>
```
Utiliser la balise Fragment exprès de React